climpred/classes.py

```diff
@@ -300,13 +297,29 @@ def __mul__(self, other):
     def __truediv__(self, other):
         return self._math(other, operator='div')
 
+    def __getitem__(self, varlist):
+        """Allows subsetting data variable from PredictionEnsemble as from xr.Dataset.
+
+        Args:
+            * varlist (list): list of name(s) of data variable(s) to subselect
+        """
+        if not isinstance(varlist, list):
```

Comment:
I would add above this:

```python
if isinstance(varlist, str):
            varlist = [varlist]
```

This ensures that a single string can be passed which will be very common and is also common in `xr.Datasets`. I.e., this allows `HindcastEnsemble['SST']` instead of `HindcastEnsemble[['SST']]`.

Reply:
but we can also add this as a fix. then both ways of accessing one variable would work. for two variables only my first implemented way works

Reply:
note `HindcastEnsemble['SST']` returns an `xr.DataArray` instead of `HindcastEnsemble[['SST']]` returning a Dataset.

Reply:
I see what you mean now. Since all of our climpred stuff expects datasets, this may be an issue. Maybe just convert it to a dataset in that case internally. Like `varlist = [[varlist]]` if that works.